internal/command/postgres/failover.go

```diff
@@ -40,6 +40,17 @@ func newFailover() *cobra.Command {
 		cmd,
 		flag.App(),
 		flag.AppConfig(),
+		flag.Bool{
+			Name:        "force",
+			Description: "Force a failover even if we can't connect to the active leader",
+			Default:     false,
+			Shorthand:   "F",
```

Comment:
A capital `F` seems a little unusual. I'd suggest either using `-f` or if that's already taken then just skip the shorthand. I think it's fine for people to write `--force`.

Reply:
i'll just skip the shorthand :)